runhouse/servers/cluster_servlet.py

```diff
@@ -214,6 +217,14 @@ async def aclear_all_references_to_env_servlet_name(self, env_servlet_name: str)
     ##############################################
     # Periodic Cluster Checks APIs
     ##############################################
+
+    def _observability_disabled(self) -> bool:
```

Comment:
the function name and response seem to be mismatched - if we're checking that observability is disabled shouldn't it just return the value of `disable_observability` in the config?

Reply:
also why do we need this function here as opposed to reusing the existing one defined in `rns_client`?